content/en/checkout/guide.md

```diff
@@ -238,6 +238,67 @@ It will reply with the capture details:
 
 If you wish to be notified each time a payment is successful, follow our [Notifications documentation](/api/notifications).
 
+### Accepting subscriptions
+
+For periodical payments with a set frequency, you can accept subscription payments in Checkout.
+
+To do so, you will need to specify a payment `type` of `["subscription"]`.
+
+You will also need to define:
+- A `payment.start_time`, that determines when the subscription payments start.
```

Comment:
Should we add here, some info about the need of the start_time being in the future?

Reply:
Yes, that's a good idea.